metrics.py

```diff
@@ -149,7 +149,9 @@ def get_stats(self):
                 errors = plugins_metadata['errors']
                 first_failed = sorted(errors.keys())[0]
                 plugins['failed_plugin'] = first_failed
-                plugins['exception'] = errors[first_failed].split("(")[0].replace(',', ' ')
+                exception_text = errors[first_failed].split("(")[0]
+                # Put exception text in quotes to allow commas in the exception
+                plugins['exception'] = '"%s"' % exception_text
```

Comment:
This will fail only if it contains unbalanced quote, it works fine with the build, which caused problems earlier (on OSD QA):
```
"<Fault 1: "<class 'pg.OperationalError'>: internal error in 'INIT': %d format: a number is required, not str">"
```